silverorange/swat
# Allow self hosted version of tinyMCE

Testing Instructions (coursehost):

1) Checkout this PR to /so/packages/swat/work-name
2) Go to /so/sites/coursehost/work-name
3) Checkout https://github.com/silverorange/course-host/pull/1752 in your coursehost directory.
4) `yarn install` to get the newest tinymce dependency
5) yarn start --symlinks=swat
6) Go to https://berna.silverorange.com/course-host-ccme/work-name/www/admin/Course/Edit?id=147 and check the WYSIWYG editor

Will require a patch release for bugfixes.
